frontend/app/routes/protected/person-case/request-details.tsx

```diff
@@ -0,0 +1,151 @@
+import { data, useFetcher } from 'react-router';
+import type { RouteHandle, SessionData } from 'react-router';
+
+import { useTranslation } from 'react-i18next';
+import * as v from 'valibot';
+
+import type { Info, Route } from './+types/request-details';
+
+import { requireAuth } from '~/.server/utils/auth-utils';
+import { i18nRedirect } from '~/.server/utils/route-utils';
+import { Button } from '~/components/button';
+import { ErrorSummary } from '~/components/error-summary';
+import { InputRadios } from '~/components/input-radios';
+import { InputSelect } from '~/components/input-select';
+import { PageTitle } from '~/components/page-title';
+import { AppError } from '~/errors/app-error';
+import { getFixedT } from '~/i18n-config.server';
+import { handle as parentHandle } from '~/routes/protected/layout';
+import { getLanguage } from '~/utils/i18n-utils';
+
+type RequestDetailsSessionData = NonNullable<SessionData['inPersonSINCase']['privacyStatement']>;
+
+/**
+ * Valid requests when creating a request
+ */
+const VALID_REQUESTS = [
+  'first-time',
+  'record-confirmation',
+  'name-change',
+  'expiry-extension',
+  'change-status',
+  'sin-confirmation',
+  'new-sin',
+] as const;
+
+/**
+ * Valid scenarios when creating a request
+ */
+const VALID_SCENARIOS = ['for-self', 'legal-guardian', 'legal-representative', 'as-employee', 'estate-representative'] as const;
+
+export const handle = {
+  i18nNamespace: [...parentHandle.i18nNamespace, 'protected'],
+} as const satisfies RouteHandle;
+
+export async function loader({ context, request }: Route.LoaderArgs) {
+  requireAuth(context.session, new URL(request.url), ['user']);
+  const t = await getFixedT(request, handle.i18nNamespace);
+  return {
+    documentTitle: t('protected:request-details.page-title'),
+    defaultFormValues: {
+      type: context.session.inPersonSINCase?.requestDetails?.type,
+      scenario: context.session.inPersonSINCase?.requestDetails?.scenario,
+    },
+  };
+}
+
+export function meta({ data }: Route.MetaArgs) {
+  return [{ title: data.documentTitle }];
+}
+
+export async function action({ context, request }: Route.ActionArgs) {
+  requireAuth(context.session, new URL(request.url), ['user']);
+  const lang = getLanguage(request);
+  const t = await getFixedT(request, handle.i18nNamespace);
+
+  const formData = await request.formData();
+  const action = formData.get('action');
+
+  switch (action) {
+    case 'back': {
+      throw i18nRedirect('routes/protected/person-case/privacy-statement.tsx', request);
+    }
+
+    case 'next': {
+      const schema = v.object({
+        scenario: v.pipe(v.picklist(VALID_SCENARIOS, t('protected:request-details.required-scenario'))),
+        type: v.pipe(v.picklist(VALID_REQUESTS, t('protected:request-details.required-request'))),
```

Comment:
No need to picklist schema in a [pipeline](https://valibot.dev/guides/pipelines/) using `v.pipe`.

```suggestion
        scenario: v.picklist(VALID_SCENARIOS, t('protected:request-details.required-scenario')),
        type: v.picklist(VALID_REQUESTS, t('protected:request-details.required-request')),
```

Reply:
See my [previous comment](https://github.com/DTS-STN/future-sir/pull/181#discussion_r1946563511).